### PR TITLE
End-to-end test using a client identifier

### PR DESCRIPTION
**Pull request by NSeydoux**

### CODE REVIEW

File: e2e/browser/test/e2e.playwright.ts [hunk 1]
```diff
@@ -96,3 +97,38 @@ test.describe("Logged In", () => {
...
+    // Enforce that we go through a full login again.
+    await app.page.context().clearCookies();
+    await app.page.fill("[data-testid=clientIdentifierInput]", "");
+    await auth.login({ allow: true });
+    await app.page.waitForSelector("span[data-testid=loggedInStatus]");
+    const failureResponse = await app.fetchResource(
+      clientAccessControl.clientResourceUrl
+    );
+    // The resource content shouldn't be available to a dynamically registered client.
+    expect(failureResponse).not.toBe(clientAccessControl.clientResourceContent);
```
Comment (chelseapinka) [inline e2e/browser/test/e2e.playwright.ts:132]: Do we want to validate what kind of response form the failure has?

File: e2e/browser/test/fixtures.ts [hunk 2]
```diff
@@ -83,6 +99,131 @@ const saveTextFile = async (options: {
...
+    throw new Error("Could not find links to the resource's ACR.");
+  }
+  const links = LinkHeaders.parse(responseLinks.toString());
+  const acrUrl = links.get("rel", "acl")[0].uri;
+  // Fetch the ACR, and add a matcher to match the given client ID.
+  const acrDataset = await getSolidDataset(acrUrl, { fetch: session.fetch });
+  const acrThing = getThing(acrDataset, acrUrl);
+  if (acrThing === null) {
+    throw new Error(
+      `The ACR ${acrUrl} doens't have a subject matching its own URL.`
```
Comment (chelseapinka) [inline e2e/browser/test/fixtures.ts:177]:
```suggestion
      `The ACR ${acrUrl} doesn't have a subject matching its' own URL.`
```
minor spelling error?

**Review by chelseapinka — APPROVED** [review]
Overall looks good to me, one or two small updates to error messages but they aren't user facing so truly not a blocker at all.

File: e2e/browser/test/e2e.playwright.ts (hunk 1 above)
Comment (NSeydoux) [reply e2e/browser/test/e2e.playwright.ts:132]: Fixed in cdfb0d2c

File: e2e/browser/test/fixtures.ts (hunk 2 above)
Comment (NSeydoux) [reply e2e/browser/test/fixtures.ts]: Aaah, me and my uncoordinated fingers.

**Comment by NSeydoux** [comment]
I'll merge https://github.com/inrupt/solid-client-authn-js/pull/2715 and cut a release, and then merge this.

**Pull request merged**